app/views/request/new.html.erb

```diff
@@ -216,7 +231,30 @@
     </div>
 
     <div id="request_form" class="request_form">
-      <%= render 'form', f: f %>
+      <% if @batch || @info_request.public_body.questions.blank? %>
+        <%= render 'form', f: f %>
+      <% else %>
+        <div id="request_form_questions">
+          <p><%= _('Are you asking for <strong>personal information</strong> that should be confidential?') %></p>
+
+          <% @info_request.public_body.questions.each do |question| %>
+            <label for="<%= question.key %>">
+              <input type="radio" name="request_form_response" id="<%= question.key %>" value="<%= question.key %>">
+              <%= _(question.text) %>
+            </label>
+          <% end %>
+        </div>
+
+        <% @info_request.public_body.questions.each do |question| %>
+          <div class="request_form_response request_form_response--<%= question.action %>" id="request_form_response--<%= question.key %>">
+            <% if question.action == :allow %>
+              <%= render 'form', f: f %>
+            <% else %>
+              <div class="request_deny_response"><%= _(question.response, public_body_name: @info_request.public_body.name) %></div>
```

Comment:
Hmm this clearly works, in that the response is rendered with the interpolated name, but I'm not sure that this will end up getting translated.

You might need to do something like I did in the `OutgoingMessage::Template` classes https://github.com/mysociety/alaveteli/blob/develop/app/models/outgoing_message/template/initial_request.rb#L21-L26.

Can probably create an issue for this as there's probably more value in getting it out early.

Reply:
Lets ditch the localisation here and bump to the theme for now. Removed in ce811f4